socorro/signature/utils.py

```diff
@@ -22,22 +22,15 @@ def get_crashing_thread(crash_data):
     """
     Takes crash data and returns the crashing thread as an int.
 
-    If it's a chrome hang (hang_type=1), then use thread 0. Otherwise, use the crashing
-    thread specified in the crash data.
-
     :arg crash_data: crash data structure that conforms to the schema
 
     :returns: crashing thread as an int
 
     """
-    crashing_thread = 0
-    if crash_data.get("hang_type", None) != 1:
```

Comment:
hang_type always ends up as 0. Thus this is always true and we can reduce it accordingly.